File: julia/morphing_data.jl
N = 10; n = 100;
q = [exp(-(i - 30)^2/100) + 2 * exp(-(i - 68)^2 / 100) for i in 1:100]
r = [exp(-(i - 50)^2/100) for i in 1:100];
q /= sum(q); r /= sum(r)

File: python/morphing_data.py
import numpy as np

N = 10
n = 100

q = [ (np.exp(-(i-30)**2 / 100 ) + 2*np.exp(-(i-68)**2 / 100)) for i in range(1,101)]
q = np.array(q)
r = np.array([np.exp(-(i - 50)**2 / 100) for i in range(1,101)])

q = q/sum(q)
r = r/sum(r)
